.github/workflows/builNTest.yaml

```diff
@@ -0,0 +1,64 @@
+name: Build and Test
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+      - develop
+
+jobs:
+  build-and-test-on-windows:
```

Comment:
I will push another file with those changes to try it out. I  was having issues with the matrix and the OS's not working together. It appeared that no runner would pick it up as it was requesting the several OS's

I also thought this may be better as it is easier to see what failed but can probably set up a way to show that

Reply:
another thought/question. Separate jobs can run parallel. If they are all the same then it may take longer to run the tests overall. on average each one takes 2.5-3.5 minutes so the test is fin less than 4 minutes. I will track this and see if it slows it down greatly

Reply:
Sam has pointed out in the docs that it will try to run these in parallel